<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncrementalBackupWindow</class>
 <widget class="QWidget" name="IncrementalBackupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IncrementalBackupTool</string>
  </property>
  <widget class="QPushButton" name="BackupButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Impact</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Incremental Backup Tool</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="BackupLocationInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Backup Location: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Backup To Restore:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Folder To Backup:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FolderToBackupInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RestoreButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BackupListBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadBackupButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Backup</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProgressStatus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
